django/example/templates/subscribe.html

```diff
@@ -43,6 +43,12 @@ <h1>{{ category.name }}</h1>
             {% endblocktrans %}
           {% endwith %}
         </p>
+        <p>
+            <b>{% trans 'Balance' %}</b>
+            {% with account_balance=account_balance %}
```

Comment:
`with` statement should be removed.

Reply:
Also, it should be wrapped into `blocktrans` tag.

Reply:
Fixed